src/client.rs

```diff
@@ -685,6 +687,65 @@ pub trait UiClient: PollClient {
     }
 }
 
+pub struct ClientBuilder<B: 'static = Empty> {
```

Comment:
`I` instead of `B`. It's a bit tricky to stay on top of Backend vs Id throught this code, I think this would help.